vispy/scene/cameras/turntable.py

```diff
@@ -97,9 +94,7 @@ def azimuth(self, azim):
 
     @property
     def roll(self):
-        """The angle of the camera in degrees around the z axis. An angle of
-        0 places puts the camera upright.
-        """
+        """Get the camera roll angle in degrees."""
```

Comment:
If the user only sees the docstring for the `getter` part of the property, should this include more information about what setting it to 0 means as it did before? If that information made it more confusing then ignore this.

Reply:
The previous information was confusing, but this information can be added for the other two angle getters.